planet/scripts/v1.py

```diff
@@ -711,8 +715,14 @@ def cancel_order(order_id, pretty):
 
 
 @click.option('--name', required=True)
-@click.option('--id', required=True,
-              help='One or more comma-separated item IDs')
+@click.option('--id', help='One or more comma-separated item IDs',
+                cls=RequiredUnless, this_opt_exists='ids_from_search')
+# Note: This is passed as a string, because --item-type is a required field for
+# both 'data search' and 'orders create'.
+@click.option('--ids_from_search',
+              help='Embedded data search')
+@click.option('--clip',
+              help='Provide a GeoJSON AOI for clipping')
```

Comment:
Clarify what you mean by AOI here. Is this a geojson Feature Collection, Feature, or Geometry? (Should we support all of those possibilities?)

Reply:
Since this client is already `click` based, [cligj](https://github.com/mapbox/cligj) could be helpful for parsing geojson args.

Reply:
it's geometry only, which is what orders supports https://developers.planet.com/docs/orders/tools-reference/#clip ... I will update the `help` message to be more clear.

also the geometry will be parsed/validated when submitted to the backend. this happens almost immediately, so I don't see too much value in also parsing it here in the client.

Reply:
clarified docs to `Geometry`